lms/templates/content_item_selection/new_content_item_selection.html.jinja2

```diff
@@ -230,8 +253,10 @@
     function pickerCallback(data) {
       var docCount = data.docs && data.docs.length;
       if (data.action == google.picker.Action.PICKED && docCount === 1) {
-        enablePublicViewing(data.docs[0]);
-        var url = buildDocUrl(data.docs[0]);
+        var doc = data.docs[0];
+        var url = buildDocUrl(doc);
+        state.selectedDocId = doc.id;
+        state.selectedDocUrl = url;
         if(url) {
```

Comment:
Should we have error handling if there is no url, or do you think the error handling in the function buildDocUrl is enough?

Reply:
Added